Examples/FOR_Loops/3_in_range.robot

```diff
@@ -5,28 +5,27 @@ Documentation    IN RANGE can get between 1 and 3 arguments.
 
 *** Test Cases ***
 Loop 10 times
-    Log To Console    start: 0 ,inc: 1 , end:10
+    Log To Console    start: 0 ,inc: 1 , end:9
     FOR    ${num}    IN RANGE    10
         Log To Console    ${num}
     END
 
 Loop 10 times starting with 1
-    Log To Console    start: 1 ,inc: 1 , end:11
+    Log To Console    start: 1 ,inc: 1 , end:10
     FOR    ${num}    IN RANGE    1    10
         Log To Console    ${num}
     END
 
 Loop starting with 0 to 100 in steps of 5
-    Log To Console    start: 1 ,inc: 1 , end:11
+    Log To Console    start: 0 ,inc: 5 , end:100
     FOR    ${num}    IN RANGE    0    101    5
         Log To Console    ${num}
     END
 
 Countdown from 10 to 0 in 10 seconds
-    Log To Console    start: 1 ,inc: 1 , end:11
-    FOR    ${num}    IN RANGE    10    -1    -1
+    Log To Console    start: 10 ,inc: -1 , end:1
+    FOR    ${num}    IN RANGE    10    0    -1
         Log To Console    ${num}
-        Exit For Loop If    ${num} == 0
         Sleep   1sec
     END
     Log To Console    🎉🥳 HAPPY NEW YEAR !!! 🍾
```

Comment:
Wenn du es bei 0 enden lässt, kommt die 0 nie.
Die Idee hier war auch das Exit bzw BREAK zu zeigen.

Ich würde es so ändern.

```robotframework
Countdown from 10 to 0 in 10 seconds
    Log To Console    start: 10 ,inc: -1 , end: -1
    FOR    ${num}    IN RANGE    10    -1    -1
        Log To Console    ${num}
        IF    ${num} == 0    BREAK
        Sleep   1sec
    END
    Log To Console    🎉🥳 HAPPY NEW YEAR !!! 🍾
```

Reply:
Kannst ja in die [Documentation] kurz reinschreiben, dass eine FOR Loop auch abgebrochen werden kann. " can conditionally be exited"